### ISSUE TITLE
Shared library error on heroku-18 stack

### ISSUE THREAD

**Issue by macowie** [dscout/wkhtmltopdf-buildpack#32]
Just noting the buildpack doesn't work as-is on the new stack.

`wkhtmltopdf: error while loading shared libraries: libXrender.so.1: cannot open shared object file: No such file or directory`

**Comment by jessamynsmith** [comment]
I found that adding the following to my Aptfile got wkhtmltopdf working again.

```
libxrender1
libssl1.0.0
```

For a working example, see this app [https://github.com/jessamynsmith/django-wkhtmltopdf-example](https://github.com/jessamynsmith/django-wkhtmltopdf-example)

**Comment by benoitongit** [comment]
Looks like libxrender1  was added in heroku-18: https://devcenter.heroku.com/articles/stack-packages.

But now getting this error: 
Command Error: /app/vendor/bundle/ruby/2.5.0/gems/wkhtmltopdf-binary-edge-0.12.5.0/libexec/wkhtmltopdf-linux-amd64: error while loading shared libraries: libpng12.so.0: cannot open shared object file: No such file or directory

Would it be fantastic if the buildpack could install all missing dependencies :)

**Comment by sorentwo** [comment]
Agreed, it would be great to have missing dependencies installed for the heroku-18 stack. A pull request is welcome!

**Comment by jathayde** [comment]
LibPNG seems to be in the list as well (not sure when it was added):
libpng12-0 | 1.2.50-1​ubuntu2.14.04.3

**Comment by lkrasovsky** [comment]
> Looks like libxrender1 was added in heroku-18: https://devcenter.heroku.com/articles/stack-packages.
> 
> But now getting this error:
> Command Error: /app/vendor/bundle/ruby/2.5.0/gems/wkhtmltopdf-binary-edge-0.12.5.0/libexec/wkhtmltopdf-linux-amd64: error while loading shared libraries: libpng12.so.0: cannot open shared object file: No such file or directory
> 
> Would it be fantastic if the buildpack could install all missing dependencies :)

do you remember how did you fix it? I have the same problem

**Comment by jessethebuilder** [comment]
This buildpack works:

https://elements.heroku.com/buildpacks/pathwaysmedical/heroku-buildpack-tesseract
